binding.gyp

```diff
@@ -56,6 +57,7 @@
             ['OS=="linux"', {
               'libraries': ['<(PRODUCT_DIR)/../../zmq/lib/libzmq.a'],
               'include_dirs': ['<(PRODUCT_DIR)/../../zmq/include'],
+              'cflags_cc': [ '-std=gnu++17' ],
```

Comment:
```suggestion
              'cflags_cc': [ '-std=c++17' ],
```

Reply:
Done! Updated Electron v21 to latest release as well.